### PR TITLE
Feature new homepage

### LINKED ISSUE #237: Home page redesign
Now that we have improved the submissions page, the next UI problem is the homepage.

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #237 

@jvillarejo @faloi @aguspina @PalumboN Please and/or make changes to this PR

This is how it looks like 

new_home

### CODE REVIEW

**Comment by flbulgarelli** [comment]
Alternatively, this is how it would look if navbar was removed:

new_home_no_nav

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,39 +1,127 @@
...
 <% end %>
 
-<div class="row">
-  <div class="col-md-4">
-    <h2 class="text-center"><%= t :mumuki_is_simple %></h2>
 
-    <p class="text-center"><%= t :mumuki_is_simple_description %></p>
-    <%= link_to t(:start_practicing), categories_path, class: 'btn btn-primary special-button btn-block', role: 'button ' %>
+<div class="mumuki-description">
+  <div class="row">
```
Comment (faloi) [inline app/views/home/index.html.erb]: A "foreach" is really needed here :stuck_out_tongue:

**Comment by faloi** [comment]
- +1 to the navbar
- The "red" button doesn't convince me, I think the previous version was better

**Comment by aguspina** [comment]
I like this new style... but im not sure of the button color. I think the previous one was better. 

+1 to the navbar too.

the description section seems like a mess to me. Maybe if all paragraphs was alligned to the left, i don't know. .I'll think a way to make it better.

**Comment by flbulgarelli** [comment]
@aguspina @faloi please propose changes or add commits to this PR as soon as possible. I would like to apply suggested changes and merge it before weekend.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (flbulgarelli) [inline app/views/home/index.html.erb]: Yeah, we should have a model named Pillar, so that we can edit them easily and avoid that mess.

**Comment by flbulgarelli** [comment]
The problem with the dark blue button is that it does not get attention. Actually, it worked, since you all get focused on it :stuck_out_tongue: 

Perhaps we could find a better color later.

**Pull request merged**